aait/backend/group-2B/Application/DTOs/PostDtos/Validators/CreatePostDtoValidator.cs

```diff
@@ -0,0 +1,30 @@
+using FluentValidation;
+using SocialSync.Application.Contracts.Persistence;
+
+namespace SocialSync.Application.DTOs.PostDtos.Validators;
+
+
+public class CreatePostDtoValidator : AbstractValidator<CreatePostDto>
+{
+    private IUserRepository _userRepository;
```

Comment:
Here, it would be better if the _userRepository marked as readonly to prevent unintentional updates and to clarify the intention.

Reply:
Fixed.